_partials/_migrate_live_migration_docker_subcommand.md

```diff
@@ -0,0 +1,93 @@
+First, set the database URIs as environment variables:
+
+```sh
+export SOURCE=""
+export TARGET=""
+```
+
+Next, download the live migration docker image:
+
+```sh
+$ docker run --rm -it --name live-migration \
+    -e PGCOPYDB_SOURCE_PGURI=$SOURCE \
+    -e PGCOPYDB_TARGET_PGURI=$TARGET \
+    --pid=host \
+    -v ~/live-migration:/opt/timescale/ts_cdc \
+    timescale/live-migration:v0.0.7 --help
+
+Live migration moves your PostgreSQL/TimescaleDB to Timescale Cloud with minimal downtime.
+
+options:
+  -h, --help            Show this help message and exit
+  -v, --version         Show the version of live-migration tool
+
+Subcommands:
+  {snapshot,clean,migrate}
+                        Subcommand help
+    snapshot            Create a snapshot
+    clean               Clean up resources
+    migrate             Start the migration
+```
+
+Live-migration contains 3 subcommands:
+1. Snapshot
+1. Clean
+1. Migrate
+
+On a high-level,
+
+the `snapshot` subcommand creates a Postgres snapshot connection to the source
+database along with a replication slot. This is pre-requisite before running
+the `migrate` subcommand.
+
+The `migrate` subcommand carries out the live-migration process by taking help
+of the snapshot and replication slot created by the `snapshot` subcommand.
+
+The `clean` subcommand is designed to remove resources related to live migration.
```

Comment:
I think we also emphasise not to run clean if they want to resume the migration.